### DISCUSSION TITLE
NPM vs NPMplus ?

### DISCUSSION THREAD

**Discussion by titro** [ZoeyVid/NPMplus#586] [Q&A] [answered]
Hi all,

i'm confused. Found this very active project as a new package in unraid. 

But also found commits from jc21 in this project. So is npm dead and this is the successor?

**Comment by Zoey2936** [comment, marked as answer]
NPM sometimes get updates, but not very often, most time (not always) they are just merges of PR which were created by users. Since I wanted to have HTTP/3 in NPM, I've forked NPM and added it and as you can see in the README, there were many other features which I've also added (darkmode, modsec, crowdsec, goaccess etc.). So NPM still sometimes get updates, those are merged into this fork, but those are most time internal changes/no new features. Also, NPM has very outdated dependencies and many CVEs. This fork still has some outdated dependencies (webpack v4, tabler v0.0.31, etc.), because updating them would be nearly like rewriting NPM. But I've tried to fix all CVEs, so there should be none at the moment. I would conclude that NPMplus is an active fork of NPM with many new features, a slimmer docker image and updates decencies,

**Reply by titro** [reply]
I was just confused, because i saw jc21 comitting and no further information in NPM oder NPMplus discussion so i was wondering if jc21 gave up his own NPM github project. 
Thanks for clarification.
